tests/src/e2e/ledger_tests.rs

```diff
@@ -945,10 +946,21 @@ fn ledger_many_txs_in_a_block() -> Result<()> {
 /// 6. Submit an invalid proposal
 /// 7. Check invalid proposal was not accepted
 /// 8. Query token balance (funds shall not be submitted)
+/// 9. Send a yay vote from a validator
+/// 10. Send a yay vote from a normal user
+/// 11. Query the proposal and check the result
+/// 12. Wait proposal grace and check proposal author funds
+/// 13. Check governance address funds are 0
 #[test]
 fn proposal_submission() -> Result<()> {
     let test = setup::network(|genesis| genesis, None)?;
 
+    let anomac_help = vec!["--help"];
+
+    let mut client = run!(test, Bin::Client, anomac_help, Some(40))?;
```

Comment:
```suggestion
    // We run `--help` to make sure that the client is compiled before we
    // start-up the ledger, so that the client commands that follow are not
    // delayed by the compilation.
    let mut client = run!(test, Bin::Client, ["--help"], Some(40))?;
```